### DISCUSSION TITLE
New planet with disease idea

### DISCUSSION THREAD

**Discussion by MyNameIsToday** [TeamGalacticraft/Galacticraft#249] [Feature Requests]
Just a suggestion ;) A planet if you stay for too long you get a disease you need to cure with the local materials and ingredients. You would have to cure yourself before returning to earth or going to other planets. Again just a suggestion, if not possible that's fine :)

**Comment by DiamondWalker** [comment]
Galacticraft planets are all based on real life, so I don't know how this would work. I guess it could be added on titan or an exoplanet, but that's a stretch.

**Comment by MyNameIsToday** [comment]
While writing this I was not completely thinking and I guess subnautica came to mind while I was writing it ;)
